CMakeLists.txt

```diff
@@ -4,6 +4,67 @@ project(mrover)
 ## Compile as C++17, supported in ROS Kinetic and newer
 add_compile_options(-std=c++17 -Wall -Wextra -Werror -pedantic)
 
+## If on raspberry pi (RPi 4 w/ 64-bit Buster)
+if (${DEVICE} MATCHES "raspi4")
+    message("Building for device: ${DEVICE}")
+
+    find_package(catkin REQUIRED COMPONENTS  
+        rospy
+        std_msgs
+        message_generation
+        dynamic_reconfigure
+    )
+    catkin_python_setup()
+
+    ## Generate messages in the 'msg' folder
+    add_message_files(
```

Comment:
I think the messages and services can be compiled by everything, take them out of here to reduce duplication.

Reply:
Will these still be needed by the Pi? Do you want these universal to the file? I'm trying to avoid adding more if/else blocks but I might have to then.